### ISSUE TITLE
Investigation into Analysis_Protocol fields

### ISSUE THREAD

**Issue by Wkt8** [ebi-ait/dcp-ingest-central#860]
Based on an in-person meeting with Dave Horsfall (Haniffa Lab) at the WSSS Conference, I have organised a meeting with him, Ida, and a couple other computational biologists in the Haniffa lab to discuss analysis_protocols, useful metadata to be captured, and compare our different schemas. 

This will give us a better understanding of what metadata is actually useful for computational biologists, and serves as a basis to inform future metadata changes to our analysis_protocol schema and information which could go into MAMS / CellXGene. 

First Meeting agenda: https://docs.google.com/document/d/1KYPaFoEF_mphRbj3cLP7oErtPXRcaSLhpRIjZ0lWR6Y/edit#

Acceptance Criteria:
- [ ] Meetings with Dave Horsfall are complete
- [x] Document writing down ideas and information with Dave Horsfall is linked here
- [ ] Tickets for future metadata-schema-changes (If any) are created and linked here

**Comment by ESapenaVentura** [comment]
Had first meeting - Second one with more people

Discuss same things, with people with more insight

**Comment by MightyAx** [comment]
waiting for response from Dave Horsfall

**Comment by ESapenaVentura** [comment]
This discussion may be coupled with #876

**Comment by Wkt8** [comment]
Dave responded and recommended the 9th and 16th of January as a next meeting date. I plan to propose a meeting time on 16th of January at 1.30pm or 2.00pm. Will cc in wrangler team email. @idazucchi would be great to have you, and @gabsie looping you in. Let me know if you would like to join.

**Comment by gabsie** [comment]
would love to be invited as an optional!

**Comment by Wkt8** [comment]
Meeting this Thursday (today)

**Comment by amnonkhen** [comment]
@Wkt8 to include summary, notes, catchup

**Comment by Wkt8** [comment]
Notes: https://docs.google.com/document/d/1vsCLzK6cOeaSPMHkMT47R9klnOfkRRDGwA8Up8ilzoM/edit

We discussed on capturing analysis protocol metadata and what metadata we should and do capture. We mentioned several differences in the entities used in the HCA schema and Hannifa lab data models (e.g. alignment).

We discussed how we update files, notifying about new versions, and saying what has changed. The HCA does not notify users when we update a file, but does publish the checksum as part of a metadata spreadsheet. 

We talked a bit about how the HCA can provide metadata in a community-informed format, such as linking out to BioSamples when applicable.
